runtime/integration-tests/Cargo.toml

```diff
@@ -9,69 +9,69 @@ repository = "https://github.com/centrifuge/centrifuge-chain"
 
 
 [dependencies]
-fudge = { git = "https://github.com/centrifuge/fudge", branch = "polkadot-v0.9.26"}
+fudge = { git = "https://github.com/centrifuge/fudge", branch = "polkadot-v0.9.28"}
```

Comment:
One problem we have is, that fudge depends on centrifuge. Currently the branch is pointing to this branch here, namely `centrifuge-runtime = {git = "https://github.com/gruberb/centrifuge-chain", branch = "polkadot-v0.9.28"}
`. 

So if you delete your branch we break our build, although only the tests as we have only dev-dependencies in fudge on centrifuge. I would suggest we merge this as is, then update the fudge branch to the rev of centriduge and crate a pr for centrifuge with the updated fudge version. 

WDYT?